index.bs

```diff
@@ -164,9 +164,8 @@ See also [[DWBP-UCR#R-FormatMachineRead]].
 
 [=Consumers=] want to refer to datasets. They prefer to do so by linking to them.
 
-> Therefore, publishers *must* maintain a permanent and unique identifier for each [=dataset=],
-> as well as their [[#organization-information|organization]] and [[#dataset-information|dataset]] information.
-> Publishers *should* use [[IRI|IRIs]] as identifiers.
+> Therefore, publishers *must* maintain a permanent and unique identifier for each [=dataset=].
+> Publishers *should* use HTTP [[IRI|IRIs]] as identifiers.
```

Comment:
Kept this as ‘should’, so this would still allow URNs. Do we want ‘must’ instead?